Python/aarch64.h

```diff
@@ -0,0 +1,50 @@
+#ifndef _AARCH_64_H_
```

Comment:
You may rename the file to Python/asm_trampoline_aarch64.h since it's very specific to Python/asm_trampoline.S.